internal/resources/resource_host.go

```diff
@@ -590,6 +600,13 @@ func resourceMetalHostRead(d *schema.ResourceData, meta interface{}) (err error)
 	d.Set(hCHAPUser, host.ISCSIConfig.CHAPUser)
 	d.Set(hCHAPSecret, host.ISCSIConfig.CHAPSecret)
 	d.Set(hInitiatorName, host.ISCSIConfig.InitiatorName)
+
+	if len(host.WWPNs) > 0 {
```

Comment:
While this was previously discussed, I'm wondering why this is needed (conditional set). without this check, hWWPNs is set to whatever host.WWPNs is

Reply:
I just accepted your suggestion on the PR as is. I was also thinking why this is needed. If you are ok I will remove this check.

Reply:
Fixed your comment.